### PR TITLE
FeliciaBeatrice PPP

### PR DESCRIPTION
**Pull request by FeliciaBeatrice**

### CODE REVIEW

**Review by ngdeqi — COMMENTED** [review]
Looks good. LGTM

File: docs/team/feliciabeatrice.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+
+- **New Feature:** Added the ability to check the information of Clients.
+  - What it does: Search through the Hash Map and show the Client's information along with the paired Property if there
+    are any.
+  - Justification: This feature allows the user to view information on a Client and the Property they are currently
+    renting (if any).
+
+
+- **Code Contributed:** [RepoSense Link](https://nus-cs2113-ay2223s1.github.io/tp-dashboard/?search=feliciabeatrice&breakdown=true&sort=groupTitle&sortWithin=title&since=2022-09-16&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other)
```
Comment (wilsonngja) [inline docs/team/feliciabeatrice.md:22]: Good job. The link seems to be working fine.

File: docs/team/feliciabeatrice.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+## Project: Property Rental Manager
+Property Rental Manager (PRM) is a desktop application that helps property agents manage, filter and monitor single
+owner rental units for appropriate tenants. This application uses Command Line Interface (CLI) and is able to display
+information quickly with minimal latency.
+
+Given below are my contributions to the project.
+- **New Feature:** Added the ability to remove Clients/Properties.
+  - What it does: Deletes the Client/Property from the Client List/Property List respectively and deletes any pairings
+    involving that Client/Property.
+  - Justification: This feature allows the user to remove any clients and properties they are no longer managing.
```
Comment (wilsonngja) [inline docs/team/feliciabeatrice.md:12]: Maybe can elaboration on the complexity about this issue?

**Review by wilsonngja — APPROVED** [review]
I think overall it LGTM. I will close this issue, but if you feel that there's a need to update the  justification, you can create a new issue. Good job :D

**Pull request merged**